blkid/probe_linux.go

```diff
@@ -0,0 +1,57 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//go:build linux
+
+package blkid
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/siderolabs/go-blockdevice/v2/blkid/internal/probers"
+)
+
+func (i *Info) probe(f *os.File, offset, length uint64) error {
```

Comment:
it just returns no information in the `Info` for now, so basically it will set some fields like block device sector size, but it won't set anything else in the info (it will be all empty)